index.js

```diff
@@ -3656,6 +3656,40 @@
       [Fn(a, b), m(a), m(a)],
       Z.sortBy);
 
+  //# zipWith :: (a -> b -> c) -> Array a -> Array b -> Array c
+  //.
+  //. Creates a new array out of the two supplied arrays
+  //. by applying the given function to each equally-positioned
+  //. pair of elements from each given array. Returns the result
+  //. of combining, pairwise, the given arrays using the given
+  //. binary function. The length of the resulting array is equal
+  //. to the length of the shorter input array.
+  //.
+  //. ```javascript
+  //. > S.zipWith(S.add, [1, 2, 3, 4], [3, 2, 1])
+  //. [4, 4, 4]
+  //.
+  //. > S.zipWith(S.add, [1, 2, 3], [3, 2, 1, 0])
+  //. [4, 4, 4]
+  //.
+  //. > S.zipWith(S.gte, [1, 2, 3], [3, 2, 1])
+  //. [true, true, false]
```

Comment:
I find this example confusing because the parameter order of `S.gte` is surprising when used this way. Let's replace all four doctests with the following two:

```javascript
> S.zipWith(s => t => s + t, ['a', 'b'], ['x', 'y', 'z'])
['ax', 'by']

> S.zipWith(s => t => s + t, ['a', 'b', 'c'], ['x', 'y'])
['ax', 'by']
```

Keep in mind that doctests and unit tests serve different roles: doctests show readers how to use a function; unit tests ensure that every code path is correct. As a result, there need not be a one-to-one correlation between a function's doctests and unit tests.

Reply:
Way better! Thank you!

Reply:
`s => t => s + t` is just S.concat, right? would you rather have S.concat or the longer one? We also have an example with `a => b > [a, b]`.

Reply:
The problem with `S.concat` is that it currently accepts its arguments one at a time or both at once. Were we to use `S.concat` here, it wouldn't be clear to readers that the function must be of type `a -> b -> c` rather than `(a, b) -> c`.

Reply:
gotcha